products/llbuildSwift/CMakeLists.txt

```diff
@@ -17,6 +17,19 @@ list(APPEND additional_args -DLLBUILD_C_API_VERSION_${LLBUILD_C_API_VERSION} -I
  
 if(APPLE)
   list(APPEND additional_args -target x86_64-apple-macosx10.10)
+else()
+  if(FOUNDATION_BUILD_DIR)
```

Comment:
Can we get a comment here explaining what these are used for (basically a link to the Swift build script, in case someone needs to refactor this).

Reply:
Done!